test/e2e/specs/editor/editor__post-advanced-flow.ts

```diff
@@ -69,7 +70,12 @@ describe( DataHelper.createSuiteTitle( `Editor: Advanced Post Flow` ), function
 			const testPage = await browser.newPage();
 			await testPage.goto( postURL.href );
 
-			await ParagraphBlock.validatePublishedContent( testPage, [ originalContent ] );
+			await ElementHelper.reloadAndRetry( testPage, loadPublishedPage );
+
+			async function loadPublishedPage(): Promise< void > {
```

Comment:
Tiny little nit: but this function name seems to me to not match what it actually does. The `reloadAndRetry` function takes care of loading the page, and this function just validates the expected content.

Reply:
Right, that was my oversight due to scope of the closure changing after I wrote it. 
Renamed it to be `validatePublishedPage`.